sw/device/silicon_creator/lib/boot_svc/boot_svc_header_unittest.cc

```diff
@@ -53,5 +53,113 @@ TEST_F(BootSvcHeaderTest, Init) {
   EXPECT_EQ(fake_msg.header.length, sizeof(fake_msg));
 }
 
+struct BootSvcTestMsg {
+  boot_svc_header_t header;
+  uint32_t payload;
```

Comment:
Nit: looks like `payload` is never initialized, but it's also never read. WDYT about changing it to `uint8_t payload[CHIP_BOOT_SVC_MSG_HEADER_SIZE + 1] = {0}` so it is large enough to accommodate the largest claimed length in `kHeaderCheckTestCases`?

Reply:
Unless I'm missing something it's fine to leave it as is since we mock digest computation and don't access the payload in the test as you said.

Reply:
Yeah, this is admittedly a nitty nit. I guess I was trying to say the existence of the `payload` field threw me off. First I noticed it wasn't initialized, but then I noticed it's never actually read.